kde-plasma/discover/discover-5.27.49.9999.ebuild

```diff
@@ -67,11 +69,18 @@ src_prepare() {
 
 src_configure() {
 	local mycmakeargs=(
+		# TODO: Port PackageKit's portage back-end to python3 and
+		# move from layman to eselect-repository
```

Comment:
Not sure why this comment needs to extend to the details of layman vs. eselect-repository, as unrealistic as it is that any porting of that backend will happen.

Reply:
Merged with a smaller comment.